app/assets/javascripts/admin/templates/user_index.hbs

```diff
@@ -225,7 +225,7 @@
 
     {{#if api_key}}
       <div class='long-value'>
-        {{api_key.key}}
+        <input type="text" value="{{api_key.key}}" style="width:450px">
```

Comment:
this is odd on a few levels, 

1. it should not be editable
2. CSS belongs in SCSS files, not inline.